tests/benchmark_test_inputs/AD_grid_diesel/csv_elements/energyProviders.csv

```diff
@@ -0,0 +1,12 @@
+,unit,DSO,Diesel
+energy_price,currency/kWh,0.54,0.1
```

Comment:
Well, 0.1 as fuel price is rather low, but yeah, for this case it makes sense!

Reply:
yes you're right, it's just to make sure LCOE_diesel is less than the grid price